application/dash_application/views/call_ready_tar.py

```diff
@@ -192,8 +192,8 @@ def get_merged_ts_data():
 # TODO: Look at ALL the reports for common labels
 cutoff_callability_label = "Callability minimum"
 initial["cutoff_callability"] = 50
-cutoff_insert_median_label = sidebar_utils.insert_median_cutoff_label
-initial["cutoff_insert_median"] = 150
+cutoff_insert_mean_label = "Insert Size Mean + Intron"
```

Comment:
Introns affect the insert size of RNA (single lane and call ready) only. Targeted sequencing and whole genome views are "Insert Size Mean".